### PR TITLE
orbiton 2.65.12

### PR DESCRIPTION
**Pull request by xyproto**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Comment by chenrui333** [comment]
@xyproto looks like https://orbiton.zip/ is down

**Comment by xyproto** [comment]
@chenrui333 Ah, yes. Thanks. I updated the formula.

File: Formula/o/orbiton.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Orbiton < Formula
   desc "Fast and config-free text editor and IDE limited by VT100"
-  homepage "https://orbiton.zip/"
-  url "https://github.com/xyproto/orbiton/archive/refs/tags/v2.65.11.tar.gz"
-  sha256 "ed0b663a91c084a05e138a1d011931a896e6a312700b984d76385e675646ddbd"
+  homepage "https://roboticoverlords.org/orbiton"
```
Comment (chenrui333) [inline Formula/o/orbiton.rb:3]:
```suggestion
  homepage "https://roboticoverlords.org/orbiton/"
```

**Comment by xyproto** [comment]
I resolved the "Commit style" by squashing and updating the branch. Hopefully that was the correct action.

**Review by chenrui333 — APPROVED** [review]

**Comment by xyproto** [comment]
Thanks for reviewing and merging.

**Pull request merged**